airflow/providers/discord/operators/discord_webhook.py

```diff
@@ -81,7 +83,7 @@ def __init__(self,
         self.proxy = proxy
         self.hook = None
 
-    def execute(self, context):
+    def execute(self, context: Dict[Any, Any]) -> None:
```

Comment:
Since it is just `Any, Any` there isn't much advantage of Dict compared to dict

```suggestion
    def execute(self, context: dict) -> None:
```